omnibus/files/chef-dk-gem-versions/default/installed_gems_as_json.rb

```diff
@@ -0,0 +1,18 @@
+# Print all the gems installed in the ChefDK omnibus as a JSON hash. This is
+# stored in the root of the ChefDK install next to the `version-manifest.json`.
+
+all_gems = {}
+
+Gem::Specification.load_defaults
+Gem::Specification.each do |spec|
+  all_gems[spec.name] ||= []
+  all_gems[spec.name] << spec.version.to_s
+end
+
+require 'json'
+j = JSON.generate(all_gems)
```

Comment:
`JSON.pretty_generate(all_gems)` would add whitespace to make the file more human readable, too.